app/commands.py

```diff
@@ -675,3 +677,48 @@ def replay_daily_sorted_count_files(file_extension):
                                                    suffix=file_extension or '.rs.txt'):
         print("Create task to record daily sorted counts for file: ", filename)
         record_daily_sorted_counts.apply_async([filename], queue=QueueNames.NOTIFY)
+
+
+@notify_command(name='populate-organisations-from-file')
+@click.option('-f', '--file_name', required=True,
+              help="Pipe delimited file containing organisation name, sector, crown, argeement_signed, domains")
+def populate_organisations_from_file(file_name):
+    # [0] organisation name:: name of the organisation insert if organisation is missing.
+    # [1] sector:: Central | Local | NHS only
+    # [2] crown:: TRUE | FALSE only
+    # [3] argeement_signed:: TRUE | FALSE
+    # [4] domains:: comma separated list of domains related to the organisation
+
+    # The expectation is that the organisation, organisation_to_service
+    # and user_to_organisation will be cleared before running this command.
+    # Ignoring duplicates allows us to run the command again with the same file or same file with new rows.
+    with open(file_name, 'r') as f:
+        for line in itertools.islice(f, 1, None):
+            columns = line.split('|')
+            print(columns)
```

Comment:
I’d add some checks to make sure the data is what we expect:
```suggestion
            print(columns)
            assert columns[0]
            assert columns[1].lower in {'central', 'local', 'nhs'}
            assert columns[2] in {'TRUE', 'FALSE', ''}
            assert columns[3] in {'TRUE', 'FALSE', ''}
```

Reply:
I don't think it's necessary - the file works as is.